ci/docker.go

```diff
@@ -50,9 +50,19 @@ func (d *Docker) Run(ctx context.Context, job *Job, user string) (string, error)
 	// will wait until the container stops
 	waitc, errc := cli.ContainerWait(ctx, resp.ID, container.WaitConditionNotRunning)
 
+	if timeout < 1 {
+		timeout = containerTimeout
+	}
+
 	select {
 	case wErr := <-errc:
+		fmt.Println("wErr: ", wErr.Error())
 		return "", wErr
+		// if the container still running after predefined time interval, force kill it
+	case <-time.After(timeout):
+		fmt.Println("Timeout done, stopping container: ", user)
```

Comment:
String should say: "Timeout, stopping container: "
(but not sure we should print like this; does Docker struct have a logger?)

Reply:
No loggers as far as I can see. Not sure we have to log it here either, the message is being saved in the build logs of the updated submission.